bundle.yaml.j2

```diff
@@ -37,26 +37,25 @@ applications:
     series: jammy
     scale: 1
     trust: true
-  tls-certificates-operator:
-    charm: tls-certificates-operator
-    channel: stable
+  self-signed-certificates:
+    charm: self-signed-certificates
+    channel: edge
```

Comment:
Shouldn't this point to a specific revision?

Reply:
ok, I'll fix it.